backend/src/modules/socket/gateway/socket.gateway.ts

```diff
@@ -20,6 +21,10 @@ export default class SocketGateway
 
 	private logger: Logger = new Logger('AppGateway');
 
+	sendUpdatedBoards(excludedClient: string) {
+		this.server.except(excludedClient).emit('updateBoardList');
```

Comment:
```suggestion
		this.server.to(roomId).except(excludedClient).emit('updateBoardList');
```

I think you should not notify all server. The `roomId` should be the same as the one sent when the user joined the room  (line 51 - payload.boards)